tests/integration/targets/connection_chroot/aliases

```diff
@@ -1,2 +1,3 @@
 needs/root
 shippable/posix/group3
+skip/macos  # FIXME
```

Comment:
Is this a temporary FIXME or just a note for you to fix it within this PR?

Reply:
I'm not sure yet. I'll probably keep it though and create an issue so someone else can fix that part. (Same for `consul`.)

Reply:
... and hashi_vault.

Reply:
Should we put here an issue id?

Reply:
The issues (#1016, #1017, #1018) are tracked in the [testing issues project](https://github.com/orgs/ansible-collections/projects/2).